string recipient
string message_title
string message_text
---
int32 result
